### ISSUE TITLE
 Association with remote system [akka.tcp://JobServer@127.0.0.1:37769] has failed - Unable to sumit jobs to spark job server (0.9.0)

### ISSUE THREAD

**Issue by Nikhith-kudumula** [spark-jobserver/spark-jobserver#1288]
**Used Spark version** - 2.4

**Used Spark Job Server version**  - 0.9.0
(Released version, git branch or docker image version)

**Deployed mode** Client

**Logs**

[2020-03-26 18:17:58,570] INFO  AkkaClusterSupervisorActor [] [] - SparkContext 0b3b9719-e47e-4a3e-a514-c4b503142065-spark.jobserver.util.TestClass joined
[2020-03-26 18:17:58,703] WARN  ReliableDeliverySupervisor [] [akka.tcp://JobServer@127.0.0.1:41048/system/endpointManager/reliableEndpointWriter-akka.tcp%3A%2F%2FJobServer%40127.0.0.1%3A37769-3] - Association with remote system [akka.tcp://JobServer@127.0.0.1:37769] has failed, address is now gated for [5000] ms. Reason: [Disassociated]
[2020-03-26 18:18:03,190] WARN  .cluster.ClusterCoreDaemon [] [akka.tcp://JobServer@127.0.0.1:41048/system/cluster/core/daemon] - Cluster Node [akka.tcp://JobServer@127.0.0.1:41048] - Marking node(s) as UNREACHABLE [Member(address = akka.tcp://JobServer@127.0.0.1:37769, status = Up)]. Node roles [supervisor]
[2020-03-26 18:18:04,188] INFO  .Cluster(akka://JobServer) [] [akka.cluster.Cluster(akka://JobServer)] - Cluster Node [akka.tcp://JobServer@127.0.0.1:41048] - Marking node(s) as REACHABLE [Member(address = akka.tcp://JobServer@127.0.0.1:37769, status = Up)]. Node roles [supervisor]

spark-job-server.out log file :

WARN remote.ReliableDeliverySupervisor: Association with remote system [akka.tcp://JobServer@127.0.0.1:41269] has failed, address is now gated for [5000] ms. Reason: [Object of unknown class cannot be deserialized spark.jobserver.io.BinaryInfo; local class incompatible: stream classdesc serialVersionUID = 552319837948673332, local class serialVersionUID = -3844678735557819007]

Getting the above error when submitting the jobs to spark job server context.

**Comment by valan4ik** [comment]
The real error seems to be:
```
Object of unknown class cannot be deserialized spark.jobserver.io.BinaryInfo; local class incompatible: stream classdesc serialVersionUID = 552319837948673332, local class serialVersionUID = -3844678735557819007
```

Could you please try newly released Jobserver 0.10.0?
I think support for Spark 2.4 was added only recently.

**Comment by Nikhith-kudumula** [comment]
Sure thank you.
